spec/diddocuments.md

```diff
@@ -184,8 +185,100 @@ Secp256k1 public keys must be converted to a verification method with a type of
 
 TODO: Add example of Secp256k1 key in a KEL and the resultant verification method
 
+##### Thresholds
+If the current signing keys threshold (the value of the `kt` field) is greater than 1, then in addition to the verification
+methods generated according to the rules in the previous sections, another verification method
+with a type of `ConditionalProof2022` will be generated in the DID document. This verification method type is defined here:
+https://w3c-ccg.github.io/verifiable-conditions/ (TODO: add proper references).
+
+The `id` property of the verification method MUST be a relative DID URL and use the AID as the value of the fragment component, e.g., `"id": "#<aid>"`.
+
+The `controller` property of the verification method MUST be the value of the `id` property of the DID document. (Does the method spec need to specify this?)
+
+The `threshold` property of the verification method MUST be the integer value of the `kt` field in the current key state.
+
+The `conditionThreshold` property of the verification method MUST contain an array. For each key listed in the array value of the `k` field in the
+key state, the relative DID URL corresponding to the key MUST be added to the array value of the `conditionThreshold` property.
+
+For example, a KERI AID with only the following inception event in its KEL:
+
+```json
+{
+  "v": "KERI10JSON0001b7_",
+  "t": "icp",
+  "d": "Ew-o5dU5WjDrxDBK4b4HrF82_rYb6MX6xsegjq4n0Y7M",
+  "i": "Ew-o5dU5WjDrxDBK4b4HrF82_rYb6MX6xsegjq4n0Y7M",
+  "s": "0",
+  "kt": "2",  // Signing Threshold
+  "k": [
+    "1AAAAg299p5IMvuw71HW_TlbzGq5cVOQ7bRbeDuhheF-DPYk",  // Secp256k1 Key
+    "DA-vW9ynSkvOWv5e7idtikLANdS6pGO2IHJy7v0rypvE",      // Ed25519 Key
+    "DLWJrsKIHrrn1Q1jy2oEi8Bmv6aEcwuyIqgngVf2nNwu"       // Ed25519 Key
+  ],
+}
+```
+
+... would result in a DID document with the following verification methods array:
+
+```json
+{
+  "verificationMethod": [
+    {
+      "id": "#Ew-o5dU5WjDrxDBK4b4HrF82_rYb6MX6xsegjq4n0Y7M",
+      "type": "ConditionalProof2022",
+      "controller": "did:webs:example.com:Ew-o5dU5WjDrxDBK4b4HrF82_rYb6MX6xsegjq4n0Y7M",
+      "threshold": 2,
```

Comment:
["1/2", "1/2", "1/2"]

Reply:
This is an example of a fractionally weighted threshold in a KERI key event

Reply:
Okay, I'm not quite sure if this can be modeled using ConditionalProof2022, but I will research it. If it can't, then we can either 1. decide that AIDs with such fractionally weighted thresholds can't (yet) be expressed as DID documents, or 2. define our own custom verification method type that is more closely based on the KERI treshold model.

Reply:
@pfeairheller @2byrds Is there an example somewhere of "kli incept" commands that will create an AID which has such fractionally weighted tresholds?

Reply:
Yes, if you look at the script `scripts/demo/basic/multisig-triple.sh` that uses the config file for inception of `scripts/demo/data/multisig-triple-sample.json` it uses a fractionally weighted threshold for both signing and rotation.

Reply:
Thanks, I'm working on this and should have an update later today.